client/src/components/ProductCard/components/ProductDescription/ProductsDescribtion.tsx

```diff
@@ -0,0 +1,72 @@
+import style from './styles.module.scss';
+import message from '@/assets/message.svg';
+import stars from '@/assets/3and5stars.png';
+import { COLORS } from '@/shared/constants/constants';
+import { TextButton } from '@/shared/components/TextButton/TextButton';
+import { IconButtonWithText } from '@/shared/components/IconButtonWIthText/IconButtonWithText';
+import cart from '@/assets/shopping-cart-white.svg';
+import cc from '@assets/cc.svg';
+import fit from '@assets/fit.svg';
+import shipping from '@assets/shipping.svg';
+import returns from '@assets/return.svg';
+import { Feature } from './components/Feature/Feature';
+
+export const ProductsDescription = () => {
```

Comment:
fix component folder name. it should be named as the component itself
ProductsDescription